src/js/entities-service/entities/groups.js

```diff
@@ -11,6 +13,45 @@ const _Model = BaseModel.extend({
   getClinicians() {
     return Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));
   },
+  addClinician(clinician) {
+    const url = `/api/groups/${ this.id }/relationships/clinicians`;
+    const group = this.toRelation(this);
+
+    clinician.set({ _groups: clinician.get('_groups').concat({ id: this.id }) }, {
+      relationships: { group },
```

Comment:
I don't think there's a need to update relationships on a set.  not sure that really does anything.  might not need `toRelation` in this method